wewatch-client/src/components/Match/Match.js

```diff
@@ -9,6 +9,7 @@ import Layout from 'views/layout';
 
 const Match = (props) => {
     const movieData = props.location.state.matchedMovie;
+    console.log(props)
```

Comment:
nit: i think we should remove logs that arent showing a changed state or event within the app